sql/hive/src/main/scala/org/apache/spark/sql/hive/client/IsolatedClientLoader.scala

```diff
@@ -55,7 +55,8 @@ private[hive] object IsolatedClientLoader extends Logging {
       sharedPrefixes: Seq[String] = Seq.empty,
       barrierPrefixes: Seq[String] = Seq.empty): IsolatedClientLoader = synchronized {
     val resolvedVersion = hiveVersion(hiveMetastoreVersion)
-    // We will use Hadoop 2.7 if we cannot resolve the Hadoop artifact.
+    // We will use hadoop-3.3.1 if we can't resolve the hadoop artifact
```

Comment:
nit: maybe `hadoop-3.3.1` -> `hadoop-3.x` so we don't need to update here every time we upgrade Hadoop 3.x version.

Reply:
Done,  Hadoop 3.x